src/lib/y2storage/clients/inst_disk_proposal.rb

```diff
@@ -45,6 +45,8 @@ class InstDiskProposal
       include InstDialogMixin
       include PartitioningFeatures
 
+      attr_reader :manual_changed # Settings has been changed by user
```

Comment:
There are already ways to ask the `StorageManager` if the staging devicegraph has been set manually. Adding another attribute for the same here doesn't sound like a good idea.

Reply:
For example, in `Clients::PartitionProposal` we have this method which achieves the same without the need of introducing a new error-prone variable:

```ruby
def manual_partitioning?
  storage_manager.proposal.nil?
end
```

Reply:
Moreover, using an instance variable in `Clients::InstDiskProposal` implies that if you go forward in the installer and back again to this screen the flag is lost. Isn't it?

I mean, if I understand the code, this will not raise the warning (although it should):

 * Reach the screen for the first time
 * Run the expert partitioner and setup the system
 * Click next to go to the next installer screen
 * Press back to return to the Storage Proposal screen
 * Press Guided Setup

Reply:
The fix of the fix is here https://github.com/yast/yast-storage-ng/pull/705

Tested to work even after going back